src/patternfly/components/Button/styles.scss

```diff
@@ -290,6 +312,71 @@
     }
   }
 
+  // Link and Action Buttons
+  &.pf-m-link,
+  &.pf-m-action {
```

Comment:
`.pf-m-action` seems confusing because I think of all buttons as actions. If the purpose is to apply the styles we want when using an icon instead of text for a button, then should it be `.pf-m-icon` instead?

Reply:
wfm!